samples/README.md

```diff
@@ -1,10 +1,95 @@
-### Running the Samples
-
-install the dependencies
-```
-$ npm install
-```
-run the sample 
-```
-$ npx ts-node filename.ts
-```
\ No newline at end of file
+# NODE-SDK sample templates
```

Comment:
Make changes to match comments in the parallel Python SDK PR: https://github.com/skyflowapi/skyflow-python/pull/64